tools/run_tests/run_tests.py

```diff
@@ -759,8 +759,12 @@ def dockerfile_dir(self):
             self.python_manager_name(), _docker_arch_suffix(self.args.arch))
 
     def python_manager_name(self):
-        if self.args.compiler in ['python3.5', 'python3.6']:
-            return 'pyenv'
+        if self.args.compiler == 'python3.5':
+            return 'stretch_35'
+        elif self.args.compiler == 'python3.6':
+            return 'stretch_36'
+        elif self.args.compiler == 'python3.7':
+            return 'stretch_37'
         elif self.args.compiler == 'python_alpine':
             return 'alpine'
         else:
```

Comment:
Where will we need `jessie`? Can we get rid of it?

Reply:
Our current test on Python3.4 is depend on `jessie`, I didn't want to touch current test logic. I will remove it.

Reply:
What prevents us from making it run on the new `stretch` images like above?

Reply:
(note that we should move our tests to 3.7.0 too. there's an open issue for it)

Reply:
The Python3.4 can only be installed by `apt-get` in `jessie`. The `stretch` minimal version of Python3 is Python3.5. So, can we put the `jessie` back for now. And once we moved to Python3.7 successfully, we remove the `jessie` dependency?